obsolete/DSharpPlus.Lavalink/DiscordClientExtensions.cs

```diff
@@ -71,9 +35,6 @@ public static Task ConnectAsync(this DiscordChannel channel, LavalinkNodeConnect
             throw new NullReferenceException();
         }
 
-        LavalinkExtension lava = discord.GetLavalink();
-        return lava == null
-            ? throw new InvalidOperationException("Lavalink is not initialized for this Discord client.")
-            : node.ConnectAsync(channel);
+        throw new NotImplementedException("DSharpPlus.Lavalink is defunct.");
```

Comment:
Could just as well delete the whole extension if we're gonna block anyone from using it anyway

Reply:
no objections - as it stands, trying to compile against it will error, and it doesn't work at all after some changes youtube made

Reply:
That's an issue for a different PR.